libretro/libretro-core.c

```diff
@@ -35,6 +35,50 @@ static char buf[64][4096] = { 0 };
 long microSecCounter=0;
 int cpuloop=1;
 
+#ifdef __WIN32__
+int numlock_original_state=0;
+
+#ifdef __WIN32__

```

Comment:
Why is there a dual Win32 check here?

Reply:
It seems this would break on all non-Win32 non-X11 platforms were the dual check removed.

Reply:
It was just as a placeholder for future additions